lib/rsk-utils.js

```diff
@@ -410,6 +412,59 @@ const findEventInBlock = async (rskTxHelper, eventName, fromBlockHashOrBlockNumb
   return null;
 };
 
+/**
+ * Gets all the bridge events in the block range from `fromBlockHashOrBlockNumber` to `toBlockHashOrBlockNumber`.
+ * If no `fromBlockHashOrBlockNumber` is provided, it will start from the latest block number up to the `toBlockHashOrBlockNumber`.
```

Comment:
if no `from` value is provided it starts from the latest? Where can it go from there?

Reply:
Updated.